extension/js/popup.js

```diff
@@ -1,7 +1,11 @@
-const url = "https://jzvkkf7p6d.execute-api.us-east-2.amazonaws.com/dev/deepcite";
-//const url = "http://localhost:5000/api/v1/deep_cite";
-//const url = "http://localhost:5000/";
-var ajax = null;
+// Initialize the Amazon Cognito credentials provider
+AWS.config.region = 'us-east-2'; // Region
```

Comment:
i'm assuming this AWS observable is being defined in the amazon script?

Reply:
I believe so. Unfortunately not sure how it all works. Just found this setup in the aws documentation.